<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrightNContrast</class>
 <widget class="QMainWindow" name="BrightNContrast">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BrightNContrast</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="OriginalView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="OverlayView">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>20</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadOriginal">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Original</string>
    </property>
   </widget>
   <widget class="QSlider" name="BrightSlider">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>22</width>
      <height>181</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="ContrastSlider">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>22</width>
      <height>181</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="brightness">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Brightness</string>
    </property>
   </widget>
   <widget class="QLabel" name="contrast">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Contrast</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TransferOriginal">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Transfer test image</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="GenerateView">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>260</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Overlay">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>0</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Overlay</string>
    </property>
   </widget>
   <widget class="QLabel" name="Generated">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Generated</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadOverlay">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Overlay</string>
    </property>
   </widget>
   <widget class="QLabel" name="Original">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Original</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TransferOverlay">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Transfer overflow image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Overlay</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
